src/ToDoList.js

```diff
@@ -135,20 +133,25 @@ const ToDoList = () => {
             >
               <ListItemIcon>
                 <Checkbox
-                  checked={task.completed}
+                  checked={task.isCompleted}
                   onChange={() => {
```

Comment:
since the function does multiple stuff, it would be better to extract its code to a named function.
also, we try to give descriptive names to the variables, so `task` or `updatedTask` instead of `t` would be better.

Reply:
Reverted to the original code with a named function.